_build/utils/eng_enhance.ini

```diff
@@ -150,6 +150,7 @@ text=/.*/;func=/advcl:relcl/;func=/.*obj/&xpos=/^W(DT|P.?)$/	#1>#2>#3	#1~#3;#3:e
 text=/.*/;func=/advcl:relcl/;func=/obl/&xpos=/^W(DT|P.?)$/	#1>#2>#3	#1~#3;#3:edep=ref;#2~#1;#1:edep=advcl
 # normal relative
 text=/.*/;func=/acl:relcl/;func=/(.*)/&xpos=/^W(DT|P.?)$/	#1>#2>#3	#1~#3;#3:edep=ref;#2~#1;#1:edep=$1
+text=/.*/;func=/acl:relcl/;func=/(.*)/&xpos=/^WRB$/	#1>#2>#3	#1~#3;#3:edep=ref;#2~#1;#1:edep=obl
```

Comment:
Thanks - why obl and not advmod?

Also: this file is stale - please pull from dev and add this rule to the current dev version or eng_enhance.ini, otherwise we will get a conflict